### ISSUE TITLE
[Compatibility] Lategame Upgrades

### ISSUE THREAD

**Issue by SKAREZ** [notnotnotswipez/MoreCompany#28]
**Hello, I don't know if I really need to do this issue on this repo, but I'll do it so you can be aware of this compatibility problem with the [Lategame Upgrades](https://github.com/Malcolm-Q/LC-LateGameUpgrades) mod.**

I've noticed that Lategame Upgrades dosen't work properly with MoreCompany, I'll be quick and clear:

Some of the items are broken:
- Peeper
  - [See this Issue from the LGU repo](https://github.com/Malcolm-Q/LC-LateGameUpgrades/issues/139)
- NV Goggles
  - Not sure if it's caused by MoreCompany, but they don't work as intended.

Some of the options are broken:
- The **"Individial Upgrade"** bool for making an upgrade to be an "Individual" (`true`) or an "For Everyone" (`false`) upgrade dosen't work.
  - If this bool is set to `false` (so "For Everyone") it will only apply the upgrade for the client that bought it, even if the bool is set to `false`.

I thinks that's all I've noticed that were broken, again, it might be a problem from LGU, I don't really know.
Anyway, thanks for your work!

**Comment by 1A3Dev** [comment]
Peeper was fixed in 1.8.0+

**Comment by SKAREZ** [comment]
Hey, I completely forgot I've opened this issue.
Indeed I've seen that the Peeper got fixed, the NV Goggles as been also fixed I guess, it was working well when I played with it days ago. And for the "Individial Upgrade" thing, not quite sure, but it seems to also work as intended now.

However I've encountered another bug, when you have upgrades such as the "Stimpack", that gives you more HP *(up to 180 if level max)* and failed a quota and got everything reset back to zero, thoses upgrades stay active. Don't know if it's caused by MC or not, but I prefer notify this here.
